client/src/pages/registrationform/Review.js

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "@emotion/styled";
+import { Link } from "react-router-dom";
+
+const ReviewCard = styled.div`
+  background: var(--card-background-color);
+  color: var(--text-color-normal);
+  display: flex;
+  flex-direction: column;
+  border: none;
+  border-radius: 5px;
+  padding: 1em 1em;
+  height: 80%;
+  grid-area: 2 / 2 / 3 / 3;
+  & > div {
+    display: flex;
+    flex-direction: column;
+    margin: 1em 0;
+    color: var(--text-color-attention);
+  }
+  & > div > div:first-child > span:first-child {
+    color: var(--text-color-attention);
+  }
+  & > div > div > a,
+  > a {
+    border: none;
+    background: transparent;
+    color: var(--button-proceed-color);
+    text-align: center;
+  }
+  & > div > div {
+    display: flex;
+    justify-content: space-between;
+    color: var(--text-color-normal);
+  }
+`;
+
+function Review({ setForm, formData }) {
+  const {
+    firstName,
+    lastName,
+    nickName,
+    birthDay,
+    userName,
+    password,
+    passwordRepeat,
+    moodstername,
+  } = formData;
+
+  if (
+    firstName === null ||
+    lastName === null ||
+    nickName === null ||
+    birthDay === null ||
+    userName === null ||
+    password === null ||
+    passwordRepeat === null ||
+    moodstername === null
+  ) {
+    return (
+      <ReviewCard>
+        <div>
+          Du scheinst Daten vergessen zu haben. Bitte schaue nochmal von Beginn
+          an die Anmeldung durch und fülle alle Felder aus.
+        </div>
+        <Link to="/user">Klick hier um alles überprüfen...</Link>
+      </ReviewCard>
+    );
+  } else if (password !== passwordRepeat) {
+    return (
+      <ReviewCard>
+        <div>Dein Passwort stimmt nicht überein. Bitte überprüfe dieses.</div>
+        <Link to="/userlogin">Bearbeiten...</Link>
+      </ReviewCard>
+    );
+  } else {
+    return (
+      <ReviewCard>
+        <h2>Lass uns deine Daten überprüfen</h2>
+        <div>
+          <div>
+            <span>Nutzerdaten</span>
+            <Link to="/user">Bearbeiten...</Link>
+          </div>
+          <div>
+            <span>Vorname:</span> <span>{`${firstName}`}</span>
```

Comment:
Happens multiple times in this file.

```suggestion
            <span>Vorname:</span> <span>{firstName}</span>
```

Reply:
Thx. WIll be corrected with the next commit.